src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,170 @@
+          Bugs
+        </Typography>
+      </div>
+      <div className={classes.spacer} />
+      <div className={classes.actions}>
+        <Tooltip title="Filter list">
+          <IconButton aria-label="Filter list">
+            <FilterListIcon />
+          </IconButton>
+        </Tooltip>
+      </div>
+    </Toolbar>
+  );
+};
+
+BugsTableToolbar.propTypes = {
+  classes: PropTypes.object.isRequired,
+  numSelected: PropTypes.number.isRequired,
+};
+
+BugsTableToolbar = withStyles(toolbarStyles)(BugsTableToolbar);
+
+const styles = theme => ({
+  root: {
+    width: '100%',
+    marginTop: theme.spacing.unit * 3,
+  },
+  table: {
+    minWidth: 1020,
+  },
+  tableWrapper: {
+    overflowX: 'auto',
+  },
+});
+
+class BugsTable extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      order: 'desc',
+      orderBy: 'lastupdate',
+      page: 0,
+      rowsPerPage: PROJECTS_PAGE_SIZE,
+    };
+  }
+
+  handleRequestSort = (event, property) => {
+    const orderBy = property;
+    let order = 'desc';
+
+    if (this.state.orderBy === property && this.state.order === 'desc') {
+      order = 'asc';
+    }
+
+    this.setState({ order, orderBy });
+  };
+
+  handleChangePage = (event, page) => {
+    this.setState({ page });
+  };
+
+  render() {
+    const { classes, data, loading, error } = this.props;
+    const { order, orderBy, rowsPerPage, page } = this.state;
+    const emptyRows =
+      rowsPerPage - Math.min(rowsPerPage, data.length - page * rowsPerPage);
+
+    return (
+      <Fragment>
+        {loading && <div>loading...</div>}
+        {error ? (
+          <div>error...</div>
+        ) : (
+          <Paper className={classes.root}>
+            <BugsTableToolbar />
+            <div className={classes.tableWrapper}>
+              <Table className={classes.table} aria-labelledby="tableTitle">
+                <BugsTableHead
+                  order={order}
+                  orderBy={orderBy}
+                  onRequestSort={this.handleRequestSort}
+                  rowCount={data.length}
+                />
+                <TableBody>
+                  {data
+                    .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                    .map(n => (
+                      <BugsTableEntry
+                        key={`${n.description}`}
+                        project={n.project}
+                        description={n.description}
+                        tag={n.tag}
+                        assignedto={n.assignedto}
+                        lastupdate={n.lastupdate}
+                      />
+                    ))}
+                  {emptyRows > 0 && (
+                    <TableRow style={{ height: 49 * emptyRows }}>
+                      <TableCell colSpan={6} />
+                    </TableRow>
+                  )}
+                </TableBody>
+              </Table>
+            </div>
+            <TablePagination
```

Comment:
I thought we discussed removing pagination...

Reply:
i removed it in other PR